src/helpers/emojis.ts

```diff
@@ -0,0 +1,32 @@
+export const EMOJIS = {
```

Comment:
Here I prefer singular but idk if there is a real convention

Reply:
you're right